projects/packages/my-jetpack/_inc/components/product-card/index.jsx

```diff
@@ -244,6 +244,15 @@ const ProductCard = props => {
 		onDeactivateStandalone();
 	}, [ slug, onDeactivateStandalone, recordEvent ] );
 
+	/**
+	 * Called when "See detailed stats" button is clicked.
+	 */
+	const onDetsailedStatsClick = useCallback( () => {
+		recordEvent( `jetpack_myjetpack_stats_card_seedetailedstats_click`, {
```

Comment:
Yeah, I was just matching the event that is already previously in use for the "View your stats" button. If I change it, we will have a break in data, where the previous tracking data on the button is recorded with the event name `jetpack_myjetpack_stats_card_seedetailedstats_click` and then all new clicks on the button from this point forward will be recorded on a new, different event name (`jetpack_myjetpack_stats_card_view_detailed_stats_click`). I think ideally we want to avoid breaking up the data, right?

Reply:
Ahh yes, single quotes instead of backticks. I just copied it over like that, so it was already like that previously, but I agree, single quotes is most commonly what we use when defining tracking events. 👍

Reply:
> Yeah, I was just matching the event that is already previously in use for the "View your stats" button. If I change it, we will have a break in data, where the previous tracking data on the button is recorded with the event name jetpack_myjetpack_stats_card_seedetailedstats_click and then all new clicks on the button from this point forward will be recorded on a new, different event name

Aha, that makes sense - I think it's good to leave it then!